xarray/tests/test_backends.py

```diff
@@ -5043,7 +5043,7 @@ def test_extract_nc4_variable_encoding_netcdf4(self):
 
     def test_extract_h5nc_encoding(self) -> None:
         # not supported with h5netcdf (yet)
-        var = xr.Variable(("x",), [1, 2, 3], {}, {"least_sigificant_digit": 2})
```

Comment:
It looks like this typo meant that the test accidentally passed!

Let's  fix the typo and mark it with `@pytest.mark.xfail` for now.